### PR TITLE
Generic RSS Reader wrt #528 & #557

### PR DESCRIPTION
**Pull request by jig08**
A sample run gives a JSON Object as:

```
{
   "Hackernews RSS Feed":[
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11910700\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://journals.aps.org/prl/abstract/10.1103/PhysRevLett.116.241103",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 13:30:33 EDT 2016",
         "Hash-Code":"1365366114",
         "Title":"Second Gravitational Wave Detected at LIGO",
         "URI":"http://journals.aps.org/prl/abstract/10.1103/PhysRevLett.116.241103"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11911116\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://bugs.chromium.org/p/chromium/issues/detail?id=468153#c164",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 14:40:03 EDT 2016",
         "Hash-Code":"1524834187",
         "Title":"Chrome has an official policy of ignoring the autocomplete=\u201coff\u201d attribute",
         "URI":"https://bugs.chromium.org/p/chromium/issues/detail?id=468153#c164"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11909543\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://www.codelitt.com/blog/my-first-10-minutes-on-a-server-primer-for-securing-ubuntu/",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 10:29:50 EDT 2016",
         "Hash-Code":"1183547025",
         "Title":"My First 10 Minutes on a Server",
         "URI":"http://www.codelitt.com/blog/my-first-10-minutes-on-a-server-primer-for-securing-ubuntu/"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11909953\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://www.youtube.com/watch?v=L5GVOFAYi8k",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 11:38:14 EDT 2016",
         "Hash-Code":"-2000177475",
         "Title":"Statistics for Hackers [video]",
         "URI":"https://www.youtube.com/watch?v=L5GVOFAYi8k"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11911211\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://www.factmag.com/2016/06/15/trent-reznoryoutube-built-on-the-backs-of-free-stolen-content/",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 14:53:41 EDT 2016",
         "Hash-Code":"-634602311",
         "Title":"Trent Reznor says YouTube is \u201cbuilt on the backs of free, stolen content\u201d",
         "URI":"http://www.factmag.com/2016/06/15/trent-reznoryoutube-built-on-the-backs-of-free-stolen-content/"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11910706\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://queue.acm.org/detail.cfm?id=2949831",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 13:31:22 EDT 2016",
         "Hash-Code":"-204674481",
         "Title":"Research for Practice: Expert-curated guides to the best of CS research",
         "URI":"http://queue.acm.org/detail.cfm?id=2949831"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11907887\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://atastypixel.com/blog/four-common-mistakes-in-audio-development/",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 04:27:13 EDT 2016",
         "Hash-Code":"1768731002",
         "Title":"Four common mistakes in audio development",
         "URI":"http://atastypixel.com/blog/four-common-mistakes-in-audio-development/"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11909111\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://research.microsoft.com/apps/mobile/publication.aspx?id=167719",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 09:18:13 EDT 2016",
         "Hash-Code":"-1862847091",
         "Title":"Why Do Nigerian Scammers Say They Are from Nigeria? (2012)",
         "URI":"http://research.microsoft.com/apps/mobile/publication.aspx?id=167719"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11908890\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://ocw.mit.edu/courses/aeronautics-and-astronautics/16-410-principles-of-autonomy-and-decision-making-fall-2010/lecture-notes/MIT16_410F10_lec20.pdf",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 08:37:36 EDT 2016",
         "Hash-Code":"1649214835",
         "Title":"Intro to Hidden Markov Models (2010) [pdf]",
         "URI":"http://ocw.mit.edu/courses/aeronautics-and-astronautics/16-410-principles-of-autonomy-and-decision-making-fall-2010/lecture-notes/MIT16_410F10_lec20.pdf"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11909548\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://github.com/taskulu/deskulu",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 10:30:21 EDT 2016",
         "Hash-Code":"-1796656352",
         "Title":"Show HN: Deskulu \u2013 Opensource knowledgebase and ticketing system",
         "URI":"https://github.com/taskulu/deskulu"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11910250\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://blogs.msdn.microsoft.com/typescript/2016/06/15/the-future-of-declaration-files/",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 12:23:17 EDT 2016",
         "Hash-Code":"1299629211",
         "Title":"The Future of TypeScript Declaration Files",
         "URI":"https://blogs.msdn.microsoft.com/typescript/2016/06/15/the-future-of-declaration-files/"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11911310\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://news.ycombinator.com/item?id=11911310",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 15:08:48 EDT 2016",
         "Hash-Code":"1727817665",
         "Title":"Pachyderm (YC W15) is hiring a lead Node.js or Rails engineer",
         "URI":"https://news.ycombinator.com/item?id=11911310"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11911303\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://kmkeen.com/maintainers-matter/",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 15:08:01 EDT 2016",
         "Hash-Code":"1260066160",
         "Title":"Maintainers Matter: The case against upstream packaging",
         "URI":"http://kmkeen.com/maintainers-matter/"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11908254\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://clickhouse.yandex/reference_en.html",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 06:06:17 EDT 2016",
         "Hash-Code":"632689310",
         "Title":"ClickHouse \u2013 high-performance open-source distributed column-oriented DBMS",
         "URI":"https://clickhouse.yandex/reference_en.html"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11908885\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://www.nytimes.com/2016/06/19/magazine/can-netflix-survive-in-the-new-world-it-created.html",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 08:36:29 EDT 2016",
         "Hash-Code":"1200624612",
         "Title":"Can Netflix Survive in the New World It Created?",
         "URI":"http://www.nytimes.com/2016/06/19/magazine/can-netflix-survive-in-the-new-world-it-created.html"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11911008\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://orangemind.io/post/expected-utility",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 14:24:12 EDT 2016",
         "Hash-Code":"-1831263949",
         "Title":"Expected Utility",
         "URI":"http://orangemind.io/post/expected-utility"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11911402\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://www.cs.yale.edu/homes/aspnes/classes/465/notes.pdf",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 15:19:10 EDT 2016",
         "Hash-Code":"-11767176",
         "Title":"Notes on Theory of Distributed Systems [pdf]",
         "URI":"http://www.cs.yale.edu/homes/aspnes/classes/465/notes.pdf"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11908704\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://medium.dgraph.io/wisemonk-a-slackbot-to-move-discussions-from-slack-to-discourse-22a53ddce78f#.5zv27u6mt",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 07:57:07 EDT 2016",
         "Hash-Code":"2108030402",
         "Title":"Show HN: Bot to steer discussions from Slack to Discourse",
         "URI":"https://medium.dgraph.io/wisemonk-a-slackbot-to-move-discussions-from-slack-to-discourse-22a53ddce78f#.5zv27u6mt"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11909092\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://graphhopper.com/blog/2016/06/15/graphhopper-routing-engine-0-7-released/",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 09:14:29 EDT 2016",
         "Hash-Code":"-392248362",
         "Title":"GraphHopper Routing Engine 0.7 Released",
         "URI":"https://graphhopper.com/blog/2016/06/15/graphhopper-routing-engine-0-7-released/"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11911306\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://www.jacobinmag.com/2016/05/crowdfunding-kickstarter-gofundme-charity-taxes/",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 15:08:19 EDT 2016",
         "Hash-Code":"-1250293468",
         "Title":"Against the Crowdfunding Economy",
         "URI":"https://www.jacobinmag.com/2016/05/crowdfunding-kickstarter-gofundme-charity-taxes/"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11907584\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://kcsrk.info/ocaml/multicore/2016/06/11/lock-free/",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 02:53:49 EDT 2016",
         "Hash-Code":"-1898094520",
         "Title":"Lock-free programming for the masses",
         "URI":"http://kcsrk.info/ocaml/multicore/2016/06/11/lock-free/"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11911522\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://docs.google.com/forms/d/18goaHGHQFFzAxEJ41XpqwEgYEJz0XQS8B6JOUaqVynw/viewform",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 15:33:21 EDT 2016",
         "Hash-Code":"1768366823",
         "Title":"Container Usage Survey \u2013 June 2016 by Kubernetes.io",
         "URI":"https://docs.google.com/forms/d/18goaHGHQFFzAxEJ41XpqwEgYEJz0XQS8B6JOUaqVynw/viewform"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11908970\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://www.anandtech.com/show/10353/investigating-cavium-thunderx-48-arm-cores",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 08:49:38 EDT 2016",
         "Hash-Code":"700325607",
         "Title":"Investigating Cavium's ThunderX: ARM Server SoC",
         "URI":"http://www.anandtech.com/show/10353/investigating-cavium-thunderx-48-arm-cores"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11907549\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://www.bbc.com/news/science-environment-36424768",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 02:44:49 EDT 2016",
         "Hash-Code":"-1280868583",
         "Title":"The mutation that famously turned moths black during Industrial Revolution",
         "URI":"http://www.bbc.com/news/science-environment-36424768"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11907749\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://jvns.ca/blog/2016/06/15/using-ltrace-to-debug-a-memory-leak/",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 03:46:11 EDT 2016",
         "Hash-Code":"-16110693",
         "Title":"Using ltrace to debug a memory leak",
         "URI":"http://jvns.ca/blog/2016/06/15/using-ltrace-to-debug-a-memory-leak/"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11908100\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://www.intel.com/content/www/us/en/nuc/nuc-kit-nuc6i7kyk-features-configurations.html",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 05:31:01 EDT 2016",
         "Hash-Code":"-1443551180",
         "Title":"Intel Launches 4k-enabled Quad Core NUC",
         "URI":"http://www.intel.com/content/www/us/en/nuc/nuc-kit-nuc6i7kyk-features-configurations.html"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11911287\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://www.economist.com/news/finance-and-economics/21700401-vanguard-has-radically-changed-money-management-being-boring-and-cheap-index-we",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 15:05:23 EDT 2016",
         "Hash-Code":"-1936283656",
         "Title":"Index we trust",
         "URI":"http://www.economist.com/news/finance-and-economics/21700401-vanguard-has-radically-changed-money-management-being-boring-and-cheap-index-we"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11908935\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://www.timqian.com/my-headline",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 08:44:47 EDT 2016",
         "Hash-Code":"1163799035",
         "Title":"Show HN: View Top 10 posts on HN, Reddit, Medium every day, week and month",
         "URI":"http://www.timqian.com/my-headline"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11907363\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"https://getchip.com/pages/chip",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 01:43:40 EDT 2016",
         "Hash-Code":"270859450",
         "Title":"CHIP $9 Computer",
         "URI":"https://getchip.com/pages/chip"
      },
      {
         "Description":"SyndContentImpl.value=<a href=\"https://news.ycombinator.com/item?id=11907803\">Comments<\/a>\nSyndContentImpl.interface=interface com.sun.syndication.feed.synd.SyndContent\nSyndContentImpl.type=text/html\nSyndContentImpl.mode=null\n",
         "Updated-Date":"null",
         "Link":"http://www.gallup.com/poll/192581/americans-confidence-institutions-stays-low.aspx",
         "RSS Feed":"https://news.ycombinator.com/rss",
         "Published-Date":"Wed Jun 15 04:01:50 EDT 2016",
         "Hash-Code":"-121439021",
         "Title":"Americans' Confidence in Institutions Stays Low",
         "URI":"http://www.gallup.com/poll/192581/americans-confidence-institutions-stays-low.aspx"
      }
   ]
}
```

### CODE REVIEW

File: src/org/loklak/harvester/HackernewsRSSReader.java [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.List;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import com.sun.syndication.feed.synd.SyndEntry;
+import com.sun.syndication.feed.synd.SyndFeed;
+import com.sun.syndication.io.SyndFeedInput;
+import com.sun.syndication.io.XmlReader;
```
Comment (jig08) [inline src/org/loklak/harvester/HackernewsRSSReader.java]: **TODO:** Have to add the `.jar`-s for `ROME` and `JDOM`

**Comment by shivenmian** [comment]
@jig08 Travis shows some of your imports don't exist. Please check travis and remove them

**Comment by jig08** [comment]
@shivenmian Yep, doing it.

**Comment by sudheesh001** [comment]
@jig08 Where is this information being written to the index and backend ?

**Comment by jig08** [comment]
@sudheesh001 As of now, it doesn't write anywhere.
Kindly point out where and how should the `JSONObject` be sent to, I shall do it accordingly, and also similarly for other crawlers I have implemented.

**Comment by sudheesh001** [comment]
@jig08 Have a look at the TwitterScraper.java file in the harvestor/ directory, `writetoIndex` and `writetoBackend`

**Comment by Orbiter** [comment]
Why would the Hackernews RSS reader be any different from a genric RSS reader?

**Comment by jig08** [comment]
@Orbiter It was just wrt #528 , I would make this into a general RSS Reader (for Hackernews, Open Source news etc.) if you believe that it is the correct approach that is being followed  :)

File: src/org/loklak/harvester/HackernewsRSSReader.java (hunk 1 above)
Comment (jig08) [inline src/org/loklak/harvester/HackernewsRSSReader.java]: Added in https://github.com/loklak/loklak_server/pull/610/commits/488f17b1b0fb1bf8a2e2e56452e582a9ea6d3c0c

**Pull request closed without merging**